File: since2020/CodeForces/1324C.cpp
/**
 *  @judge CodeForces
 *  @id 1324C
 *  @name Frog Jumps
 *  @contest Codeforces Round #627
 * 
 *  @tag Greedy, Minimax Problem
 */
#include <bits/stdc++.h>
using namespace std;

string s;

int solve() {
	int pos, ans;
	pos = ans = 0;
	for (int i = 0; i < s.size(); i++) {
		if (s[i] == 'R') {
			ans = max(ans, i + 1 - pos);
			pos = i + 1;
		}
	}
	return max(ans, (int)s.size() + 1 - pos);
}

int main() {
	int t;
	for (cin >> t; t; t--) {
		cin >> s;
		cout << solve() << endl;
	}
}

File: since2020/CodeForces/1324C.py
'''
	@judge CodeForces
	@id 1324C
	@name Frog Jumps
	@contest Codeforces Round #627
	
	@tag Greedy, Minimax Problem
'''
from sys import stdin

input()
for line in stdin:
	s = line.strip()
	arr = [0] + [ idx + 1 for idx, ch in enumerate(s) if ch == 'R' ] + [ len(s) + 1 ]
	it = iter(arr)
	next(it)
	ans = max(map(lambda p: p[1] - p[0], zip(iter(arr), it)))
	print(ans)